cedar-policy-cli/sample-data/sandbox_a/schema.cedarschema

```diff
@@ -0,0 +1,15 @@
+// Entity type declarations
+entity Video in [Account, Album];
+entity User in [UserGroup];
+entity UserGroup;
+entity Administrator;
+entity Photo in [Account, Album];
+entity Album in [Account];
+entity Account;
+
+
+// Action declarations
+action listPhotos
+  appliesTo { principal: [User], resource: [Album, Photo, Video] };
+action view, delete, edit
+  appliesTo { principal: [User], resource: [Photo, Video, Album] };
```

Comment:
Do we want/need the comments "Entity type declarations" and "Action declarations"?  (here and in the other schemas)

Reply:
Fixed